File: src/main/java/com/back/together02be/Together02BeApplication.java
package com.back.together02be;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.scheduling.annotation.EnableAsync;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableJpaAuditing
@EnableScheduling
@EnableCaching
@EnableAsync
@ConfigurationPropertiesScan
public class Together02BeApplication {

	public static void main(String[] args) {
		SpringApplication.run(Together02BeApplication.class, args);
	}

}

File: src/main/kotlin/com/back/together02be/Together02BeApplication.kt
package com.back.together02be

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.scheduling.annotation.EnableAsync
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableJpaAuditing
@EnableScheduling
@EnableCaching
@EnableAsync
@ConfigurationPropertiesScan
class Together02BeApplication

fun main(args: Array<String>) {
    runApplication<Together02BeApplication>(*args)
}
